.github/workflows/sonar-scan-pr-status-check.yml

```diff
@@ -0,0 +1,71 @@
+name: ✅Sonar Scan Status Check
+
+
+defaults:
+  run:
+    shell: pwsh
+
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - develop
+      - preview/v*.*.*-preview.*
+      - release/v*.*.*
+      - master
+
+
+jobs:
+  sonar_analyze_code:
+    name: Analyze Code
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: 11
+          distribution: 'zulu' # Alternative distribution options are available.
+
+      - uses: actions/setup-dotnet@v3
+        name: Setup dotnet
+        with:
+          dotnet-version: '7.x.x'
+
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
+
+      - name: Cache SonarCloud packages
+        uses: actions/cache@v3
+        with:
+          path: ~/sonar/cache
+          key: ${{ runner.os }}-sonar
+          restore-keys: ${{ runner.os }}-sonar
+
+      - name: Cache SonarCloud scanner
+        id: cache-sonar-scanner
+        uses: actions/cache@v3
+        with:
+          path: ./.sonar/scanner
+          key: ${{ runner.os }}-sonar-scanner
+          restore-keys: ${{ runner.os }}-sonar-scanner
+
+      - name: Install SonarCloud scanner
+        if: steps.cache-sonar-scanner.outputs.cache-hit != 'true'
+        run: |
+          New-Item -Path ./.sonar/scanner -ItemType Directory
+          dotnet tool update dotnet-sonarscanner --tool-path ./.sonar/scanner
+
+      - name: Build and Analyze
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        run: |
+          ./.sonar/scanner/dotnet-sonarscanner begin /k:"KinsonDigital_Carbonate" /o:"kinsondigital" /d:sonar.login="${{ secrets.SONAR_TOKEN }}" /d:sonar.host.url="https://sonarcloud.io"
+
+          dotnet clean "${{ github.workspace }}/Carbonate/Carbonate.csproj";
+          dotnet build "${{ github.workspace }}/Carbonate/Carbonate.csproj" -c Debug;
+
+          ./.sonar/scanner/dotnet-sonarscanner end /d:sonar.login="${{ secrets.SONAR_TOKEN }}"
+                    
```

Comment:
Please remove the empty whitespace on the single line at the bottom of the file.  You need to remove any whitespace but leave the single empty line behind.

```suggestion

          dotnet clean "${{ github.workspace }}/Carbonate/Carbonate.csproj";
          dotnet build "${{ github.workspace }}/Carbonate/Carbonate.csproj" -c Debug;

          ./.sonar/scanner/dotnet-sonarscanner end /d:sonar.login="${{ secrets.SONAR_TOKEN }}"

```

**VSCode Sample:**
![image](https://github.com/KinsonDigital/Carbonate/assets/105235056/bea78fe3-4f4f-4d1d-ae9f-1c7d28fa9acc)